### PR TITLE
fix: connect to dashboard page when there are no existing projects

### PR DESCRIPTION
**Pull request by ryanthemanuel**
- Closes UNIFY-1065

### User facing changelog

If there are no existing projects on the connect to dashboard page, choose an existing project will be hidden.

### Additional details

UNIFY-1065 brings up a situation where the current project does not have a project id and there are no existing projects for the selected organization. When that happens and the user selects "Choose an existing project" nothing happens (since there's no list of projects to display to connect to). Instead of letting nothing happen, we will not allow the user to even select "Choose an existing project" in that scenario.

In addition to "Choose an existing project" not working correctly, the user has the choice to "Connect project" rather than "Create project" which is more appropriate in this case. Additionally this button does nothing as well. We will fix this so that we fall into the "Create project" workflow in this scenario.

### PR Tasks

- [X] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [na] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Review by elevatebart — DISMISSED** [review]

**Review by tbiethman — APPROVED** [review]

File: packages/app/src/runs/modals/SelectCloudProjectModal.vue [hunk 1]
```diff
@@ -85,6 +85,7 @@
             </span>
           </label>
           <a
+            v-if="projects.length > 0"
```
Comment (tbiethman) [inline packages/app/src/runs/modals/SelectCloudProjectModal.vue:88]: Is this check ever falsy? If the length is 0, it doesn't look like this branch of markup would be generated.

Comment (tbiethman) [reply packages/app/src/runs/modals/SelectCloudProjectModal.vue:88]: Do'h, Ryan filled me in. newProject is a `ref` and not `computed`, so this all works as expected.

**Review by tbiethman — DISMISSED** [review]

File: packages/app/src/runs/modals/SelectCloudProjectModal.cy.tsx [hunk 2]
```diff
@@ -40,5 +40,15 @@ describe('<SelectCloudProjectModal />', () => {
     cy.contains('a', defaultMessages.runs.connect.modal.selectProject.createNewProject).click()
     cy.findByLabelText(`${defaultMessages.runs.connect.modal.selectProject.projectName }*${defaultMessages.runs.connect.modal.selectProject.projectNameDisclaimer}`)
     .should('have.value', 'Test Project')
+
+    cy.contains('a', defaultMessages.runs.connect.modal.selectProject.chooseExistingProject).should('be.visible')
+  })
+
+  it('can only create new projects if the organization has no projects', () => {
```
Comment (elevatebart) [inline packages/app/src/runs/modals/SelectCloudProjectModal.cy.tsx:47]:
```suggestion
  it('can only choose an existing project if the organization has a project', () => {
```

**Review by elevatebart — CHANGES_REQUESTED** [review]

**Comment by elevatebart** [comment]
One heavy suggestion...

**Review by elevatebart — APPROVED** [review]

**Pull request merged**